Src/FluentAssertions/Common/Guard.cs

```diff
@@ -54,6 +56,22 @@ public static void ThrowIfArgumentIsOutOfRange<T>([ValidatedNotNull] T value, st
             }
         }
 
+        public static void ThrowIfArgumentContainsNull<T>([ValidatedNotNull] IEnumerable<T> values, string paramName)
```

Comment:
I don't think the attribute should be applied to these methods or the enum ones. 

The purpose of them is to aid some one analyzer in verifying that parameters in public methods are null checked.

As I recall the usage of the attribute is "when this method returns (without excepting), the variable bound to the decorated parameter is verified to be non null".

Reply:
Ah ok, I'll clean them up.